src/render/Renderer.cpp

```diff
@@ -2099,8 +2099,9 @@ void CHyprRenderer::recheckSolitaryForMonitor(PHLMONITOR pMonitor) {
     if (!PCANDIDATE->opaque())
         return;
 
-    if (PCANDIDATE->m_vRealSize->value() != pMonitor->vecSize || PCANDIDATE->m_vRealPosition->value() != pMonitor->vecPosition || PCANDIDATE->m_vRealPosition->isBeingAnimated() ||
-        PCANDIDATE->m_vRealSize->isBeingAnimated())
+    if (!PCANDIDATE->m_bIsX11 &&
```

Comment:
this check is correct and should work on x11 too

Reply:
It doesn't account for the disabled scaling for the X11 windows this way, if the scaling for the monitor is different than `1`.

Reply:
real* are logical.

Reply:
You are right, should work on x11 too, I'll remove this.